src/main/scala/cromwell/engine/db/package.scala

```diff
@@ -12,7 +12,12 @@ package object db {
   }
 
   // Uniquely identify an entry in the execution table
-  case class ExecutionDatabaseKey(fqn: FullyQualifiedName, index: ExecutionIndex)
+  case class ExecutionDatabaseKey(fqn: FullyQualifiedName, index: ExecutionIndex) {
+    def isCollector(keys: Traversable[ExecutionDatabaseKey]): Boolean = index.isEmpty &&
```

Comment:
I think if you're going to schmear the boolean across multiple lines like this ya might as well do the `= { ... }` syntax and put the `isEmpty` down a line. Or really at that point the whole expression would probably fit on one line (i.e. 3 lines total)